content/docs/fairing/azure/az-kubeflow-notebook.md

```diff
@@ -0,0 +1,186 @@
++++
+title = "Train and Deploy on Azure from a Kubeflow Notebook"
+description = "Use Kubeflow Fairing to train and deploy a model on Azure Kubernetes Service (AKS) from a notebook that is hosted on Kubeflow"
+weight = 45
++++
+
+This page documents how to run the [Fairing prediction example
+notebook][xgb-notebook] on [Azure Kubernetes Service
+(AKS)][az-kubernetes] in a notebook hosted on Kubeflow.
+
+## Prerequisites
+
+Before you configure and use the Kubeflow Fairing Python SDK, you must have a
+Kubeflow environment installed. The example notebook does in-cluster training.
+The example notebook has been tested to work in a cluster with 2 nodes and
+virtual machines with Standard_D4_v3 size. It is not recommended to use the
+smallest size virtual machines.
+
+* If you do not have a Kubeflow installation on Azure, follow the [installation
+  guide][kf-az-deploy].
+* You must have the `kubectl` command line interface installed and configured
+  to use the Kubernetes cluster where Kubeflow is installed. In the preceding
+  installation guide, the command `az aks get-credentials` configures `kubectl`
+  to access your Kubernetes cluster.
+
+## Create Azure Container Registry and Storage
+
+In this example notebook, Azure Container Registry (ACR) is used to host docker
```

Comment:
Recommendation: Avoid passive voice, here and below. An example rephrasing might be:

"The example notebook uses Azure Container Registry (ACR) to host docker ..."

Reply:
Thanks for pointing that out!